examples/website/src/server/render.tsx

```diff
@@ -54,19 +51,21 @@ export async function renderApp({
     <LazyCapture rendered={renderedLazyPaths}>
       <AppContextProvider canonicalHost={canonicalHost}>
         <ApolloProvider client={client}>
-          <HelmetProvider context={helmetContext}>
-            <StyleProvider renderer={styleRenderer}>
-              <FacebookProvider sdkLanguage={'de_DE'}>
-                <InstagramProvider>
-                  <TwitterProvider>
-                    <RouteProvider initialRoute={initialRoute}>
-                      <App />
-                    </RouteProvider>
-                  </TwitterProvider>
-                </InstagramProvider>
-              </FacebookProvider>
-            </StyleProvider>
-          </HelmetProvider>
+          <AuthProvider>
+            <HelmetProvider context={helmetContext}>
+              <StyleProvider renderer={styleRenderer}>
+                <FacebookProvider sdkLanguage={'de_DE'}>
```

Comment:
Should `de_DE` really be hard-coded or rather depending on the user's browser-settings?

Reply:
was already in the code